src/main/java/uk/gov/hmcts/reform/civil/service/DeadlinesCalculator.java

```diff
@@ -39,6 +39,11 @@ public LocalDateTime plus14DaysAt4pmDeadline(LocalDateTime startDate) {
         return calculateFirstWorkingDay(notificationDeadline).atTime(END_OF_BUSINESS_DAY);
     }
 
+    public LocalDateTime plus14DaysDeadline(LocalDateTime startDate) {
```

Comment:
please add unit test for this helper method

Reply:
test case added